### PR TITLE
Fix card layout in Dev UI

### PR DESCRIPTION
**Pull request by netodevel**
always keep the guide button on the right side.

**Before**

antes

**After**

after

### CODE REVIEW

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html [hunk 1]
```diff
@@ -2,10 +2,16 @@
 <div class="col mb-4">
     <div class="card h-100">
         <div class="card-header">
-            {it['name']}
-            {#if it['metadata'] && it['metadata']['guide']}
-            <a href="{it['metadata']['guide']}" class="text-muted float-right" title="Guide"><i class="fa fa-book"></i></a>
-            {/if}
+            <div class="row">
+                <div class="col-md-10">
```
Comment (gsmet) [inline extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html:6]: Why not 11?

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
+                    {it['name']}
+                </div>
+                <div class="col-md-1">
```
Comment (gsmet) [inline extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html:9]: From what I can see, the icon is significantly more to the left. Could we adjust that with some alignment and padding?

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! I added a couple of questions/comments.

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html (hunk 2 above)
Comment (NetoDevel) [reply extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html:9]: padding done

after-style

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html (hunk 1 above)
Comment (NetoDevel) [reply extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html:6]: I don't know why, but if I put 11, the button guide ends up going down.
I think it ends up leaving the limits of the bootstrap grid.

**Review by gsmet — APPROVED** [review]
Looks, good thanks!

**Comment by gsmet** [comment]
Yes, thanks!

**Pull request merged**